Creates AI Assistant use cases section (#5303)
elastic/security-docs: docs/assistant/llm-connector-guides.asciidoc

[[llm-connector-guides]]
= Large language model (LLM) connector setup guides

> This section contains instructions for setting up LLM connectors in {elastic-sec}. Setting up this type of connector enables you to use <<security-assistant, Elastic AI Assistant>> and <<attack-discovery, Attack discovery>>. 

Reviewer: ```suggestion
This section contains instructions for setting up LLM connectors in {elastic-sec} so you can use <<security-assistant, Elastic AI Assistant>> and <<attack-discovery, Attack discovery>>. 
```
